test/test_train_imagenet.py

```diff
@@ -164,10 +166,13 @@ def test_loop_fn(model, loader, device, context):
     return correct / total_samples
 
   accuracy = 0.0
+  writer = SummaryWriter(log_dir='/tmp/imagenet_tensorboard')
   for epoch in range(1, FLAGS.num_epochs + 1):
     model_parallel(train_loop_fn, train_loader)
     accuracies = model_parallel(test_loop_fn, test_loader)
-    accuracy = sum(accuracies) / len(accuracies)
+    print("Epoch: {}".format(epoch))
+    accuracy = mean(accuracies)
+    writer.add_scalar('Accuracy/test', accuracy, epoch)
```

Comment:
Can we add additional scalar metrics such as loss, examples/sec, etc (where the x-axis is global step number)? Similar to the tensorboard metrics we see with TF estimator. We can do this in a followup PR maybe?

Reply:
I'll be adding more in some followup PRs. Still deciding on a clean way to do the others since we'll want to average over all devices during the training loop. Or we could have N different lines on the loss curves where N=num_cores